specification/search/data-plane/Azure.Search/indexreadme.md

```diff
@@ -0,0 +1,480 @@
+# SearchIndexClient
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for SearchIndexClient.
+
+
+---
+## Getting Started
+
+To build the SDK for SearchIndexClient, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+### Basic Information
+These are the global settings for SearchIndexClient.
+
+``` yaml
+openapi-type: data-plane
+tag: package-2019-05
+```
+
+
+### Tag: package-2019-05-searchservice-preview
```

Comment:
You shouldn't need the searchservice tags in this file.

Reply:
This is only for quick development. After preview 1, I will make efforts to merge two  into readme.md. Will address this if time available.